CONTRIBUTING.md

```diff
@@ -40,5 +40,47 @@ Python code is expected to follow
 [PEP8](https://www.python.org/dev/peps/pep-0008/) and
 [not commit atrocities](https://www.youtube.com/watch?v=wf-BqAjZb8M).
 JavaScript, CSS/Less, and markup should follow our
-[front-end standards](https://github.com/cfpb/front-end).
+[front-end standards](https://github.com/cfpb/development).
 When in doubt, mimic the styles and patterns in the existing codebase.
+
+### Browser support
+
+We configure our style processor to support the following list of browsers.
+
+- Latest 2 releases of all browsers including Chrome, Firefox, Safari,
```

Comment:
This would be more readable in a list format

Reply:
Fixed with a slight change to content org that I think is cleaner and easier to understand.